src/api-reference/direct-connects/hotel-service-2/Search.markdown

```diff
@@ -36,7 +36,7 @@ The criterion is used to define the search criteria.  Currently we support only
 
 | Element       | Required | Data Type | Description |
 |---------------|----------|-----------|-------------|
-| Position      | Y        | Complex   | Used to specify the geographic coordinates of a location, expressed in notation specified by ISO standard 6709. |
+| Position      | Y*       | Complex   | Used to specify the geographic coordinates of a location, expressed in notation specified by ISO standard 6709. **Required for Search request only, but optional for Availability request!** |
```

Comment:
Please remove the exclamation point as documentation should not need the emphatic voice.

Reply:
Thanks for the info @charlieconcur, done.